monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -184,10 +202,19 @@ class ReportPageComponent extends AuthComponent {
 
   componentDidUpdate(prevProps) {
     if (this.props.report !== prevProps.report) {
-      this.setState({report: this.props.report})
+      this.updateReport(this.props.report);
```

Comment:
This is not ideal. What happens if credentials take longer to return than the report?